TS - Reorder the Cameos

MustaphaTR opened the pull request:
Reordered them to be better in playability and symetric between sides as much as possible so shortcuts can be used more easily.

Vehicle tabs may need some edits, i couldn't find something better as Nod has more vehicles than GDI.

![gdi](https://cloud.githubusercontent.com/assets/7933210/24589571/dd12c356-17dc-11e7-9f11-707269f79aaf.png)
![nod](https://cloud.githubusercontent.com/assets/7933210/24589572/dd5ea3e8-17dc-11e7-9ede-6b5dbfa39b39.png)

MustaphaTR committed:
Reorder the TS Cameos

pchote commented:
Looks great!

A couple of ideas so that icons are grouped a bit closer to the tech progression (before radar, after radar, tech center):
* Switch GDI medic and Jumpjet.
* Switch GDI hover mrls and amphib. apc.
* Cycle Nod sensor array to where the artillery is (to match GDI), then artillery to where sapc is, and sapc to where the sensory array is.

Another inconsistency I just noticed, not direct related to this PR, is that the Waste Refinery (on the buildings tab) depends on the Missile Silo (on the support tab), which feels wrong.  We may want to move that to the support tab too?

reaperrr commented:
> We may want to move that to the support tab too? Maybe the stealth generator too.

Agreed.

MustaphaTR commented:
Updated. Did the stuff @pchote said incluing moving Stealth Generator and Weed Refinery to Defence. I moved their code to `nod-support.yaml` from `nod-structures.yaml` too.
